solutions/cf-786C

```diff
@@ -0,0 +1,95 @@
+---
+id: cf-786C
+source: CF
+title: Till I Collapse
+author: Dustin Miao
+---
+
+## Solution
```

Comment:
maybe add an `Intended Solution` with a `see official analysis` and then `Alternate Solution - DNQ` for this one